### PR TITLE
Adapt Mui button styles

### PR DESCRIPTION
**Pull request by drskullster**
Change-type: minor

### CODE REVIEW

File: src/components/Form/Form.stories.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import type { Meta, StoryObj } from '@storybook/react';
 import { RJSForm } from '.';
 
 const meta = {
-	title: 'Mui Components/Form',
+	title: 'Other/RJSX Form',
```
Comment (myarmolinsky) [inline src/components/Form/Form.stories.tsx:5]:
```suggestion
	title: 'Other/RJS Form',
```

Comment (myarmolinsky) [reply src/components/Form/Form.stories.tsx:5]: It's react-json-schema-form, the JS are not intended to stand for javascript or JSX 😅

Comment (drskullster) [reply src/components/Form/Form.stories.tsx:5]: haha nice catch @myarmolinsky :)

File: src/theme.ts [hunk 2]
```diff
@@ -429,24 +433,227 @@ export const theme = createTheme({
...
+							backgroundColor: color.bg.danger.value,
+						},
+						'&.Mui-selected': {
+							backgroundColor: color.bg.danger.value,
+						},
+					},
 				},
+				// selected: {
+				// 	backgroundColor: color.bg.value,
+				// },
```
Comment (JSReds) [inline src/theme.ts:656]: we can probably remove it

**Review by JSReds — APPROVED** [review]

**Pull request merged**